.github/workflows/cicd.yml

```diff
@@ -105,7 +105,7 @@ jobs:
           image: gcr.io/${{ secrets.PROJECT_ID }}/app
           region: us-central1
           env_vars: "OPEN_QUAD_WORD=${{ secrets.OPEN_QUAD_WORD }}"
-          flags: |
+          flags: >
```

Comment:
cloud run has a bug in 0.10.0. I've been pinning to 0.9.0 until they fix it.

https://github.com/google-github-actions/deploy-cloudrun/issues/341

Reply:
Thanks for the heads up.